packages/eslint-plugin-mdx/src/processors/options.ts

```diff
@@ -0,0 +1,58 @@
+/**
+ * based on @link https://github.com/sveltejs/eslint-plugin-svelte3/blob/master/src/processor_options.js
+ */
+
+import { ProcessorOptions } from './types'
+
+export const processorOptions = {} as ProcessorOptions
+
+// find Linter instance
+const linterPath = Object.keys(require.cache).find(
+  path =>
+    path.endsWith('/eslint/lib/linter/linter.js') ||
+    path.endsWith('\\eslint\\lib\\linter\\linter.js'),
+)
```

Comment:
this seems a bit fragile?
Would it make sense to use something like `require.resolve`?

Reply:
I'm not sure, it's based on https://github.com/sveltejs/eslint-plugin-svelte3/blob/master/src/processor_options.js

Reply:
If we use `require.resolve`, it will always be added into `require.cache`, so I don't know is that correct.

Reply:
I tried:

```ts
let resolvedLinterPath: string

// find Linter instance
const linterPath = Object.keys(require.cache).find(
  path =>
    path === resolvedLinterPath ||
    (resolvedLinterPath = require.resolve('eslint/lib/linter/linter')),
)
```

And there are an error:

```log
(node:46681) Warning: Accessing non-existent property 'Linter' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)

Oops! Something went wrong! :(

ESLint: 7.21.0

TypeError: Cannot read config file: /Users/JounQin/Workspaces/GitHub/eslint-mdx/node_modules/@1stg/eslint-config/base.js
Error: Cannot read property 'prototype' of undefined
Referenced from: /Users/JounQin/Workspaces/GitHub/eslint-mdx/node_modules/@1stg/eslint-config/index.js
    at Object.<anonymous> (/Users/JounQin/Workspaces/GitHub/eslint-mdx/packages/eslint-plugin-mdx/src/processors/options.ts:33:27)
    at Module._compile (/Users/JounQin/Workspaces/GitHub/eslint-mdx/node_modules/v8-compile-cache/v8-compile-cache.js:192:30)
    at Module.m._compile (/Users/JounQin/Workspaces/GitHub/eslint-mdx/node_modules/ts-node/src/index.ts:1056:23)
    at Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Object.require.extensions.<computed> [as .ts] (/Users/JounQin/Workspaces/GitHub/eslint-mdx/node_modules/ts-node/src/index.ts:1059:12)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (/Users/JounQin/Workspaces/GitHub/eslint-mdx/node_modules/v8-compile-cache/v8-compile-cache.js:159:20)
    at Object.<anonymous> (/Users/JounQin/Workspaces/GitHub/eslint-mdx/packages/eslint-plugin-mdx/src/processors/remark.ts:6:1)
```